src/factories/StandardPythFactory.sol

```diff
@@ -38,7 +38,7 @@ contract OvalPyth is MutableUnlockersController, PythSourceAdapter, ChainlinkDes
  * needed if different mutability or interface choices are desired.
  */
 contract StandardPythFactory is Ownable, BaseFactory {
-    IPyth immutable pyth;
+    IPyth public immutable pyth;
```

Comment:
some inconsistency here: in the `src/factories/StandardCoinbaseFactory.sol` the `public immutable` variable is all caps. here, its not. This should be standardized. same goes for the `src/oracles/CoinbaseOracle.sol`